content/resources.md

```diff
@@ -15,68 +15,84 @@ weight: 20
 
 # Course
   * We are currently studying the material from {{< courselink >}}.
+  ---
```

Comment:
I'm pretty sure the update to this file was unintentional :-)

The preview shows the effect on the *Resources* page: https://deploy-preview-87--pythonpirates.netlify.com/resources/

Reply:
Yep, that was an accident. Is there a way to just update with my new post but not change that file? Is that something I should do by fixing my local copy and submitting a pull request again, or can we just pick what parts to update here on github?

Reply:
So long as this PR remains open, any changes you make to your branch on your fork will be reflected here.

Undo the change to `resources.md` on your local branch `next-week-goals` and push it to your fork.  Once that's done this PR will update to show only **1** file change.

This should revert the file content:
```
git checkout origin/master content/resources.md

git add content/resources.md

git commit -m "restored resources"
```